src/praisonai-agents/tests/unit/tools/test_subagent_tool.py

```
@@ -256,6 +256,158 @@ def chat(self, prompt):
+
+        mgr = BackgroundJobManager()
+        fired = []
+
+        def boom():
+            raise RuntimeError("kaboom")
+
+        job_id = mgr.start_job(boom, on_complete=lambda info: fired.append(info))
+        try:
+            mgr.get_result(job_id)
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)